{"name": "test_demo", "status": "failed", "statusDetails": {"message": "AssertionError: False is not true", "trace": "self = <unit.test_demo.TestDemo testMethod=test_demo>\n\n    def test_demo(self):\n        print(\"testdemo\")\n>       self.assertTrue(False)\nE       AssertionError: False is not true\n\nunit/test_demo.py:32: AssertionError"}, "attachments": [{"name": "stdout", "source": "697c68e1-5413-4286-b36b-9a49d4dc7dc0-attachment.txt", "type": "text/plain"}], "start": 1584177260502, "stop": 1584177260502, "uuid": "6e2051e1-6663-4f9b-84aa-2e1db248f47b", "historyId": "30c1e5a57ae333ffbe92bfa2fcc203b6", "testCaseId": "609a731e35f5ad87b7ed57d5b94a1d0f", "fullName": "unit.test_demo.TestDemo#test_demo", "labels": [{"name": "parentSuite", "value": "unit"}, {"name": "suite", "value": "test_demo"}, {"name": "subSuite", "value": "TestDemo"}, {"name": "host", "value": "chloehandeMacBook-Pro.local"}, {"name": "thread", "value": "21269-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "unit.test_demo"}]}